src/components/views/onboarding/permissions.tsx

```diff
@@ -48,6 +48,7 @@ export const Permissions: FC<any> = () => {
   const handleRegistration = async (skip: boolean) => {
     try {
       app.showActivityIndicator();
+      const analyticsConsent = true;
```

Comment:
True? Why true? Is there somewhere where it says the user accepts this and I've missed that part? I know this was the case, but should we challenge it?

Reply:
@floridemai are you satisfied with Shaun's explanation of https://github.com/project-vagabond/covid-green-app/pull/39 on this?